exec/java-exec/src/main/java/org/apache/drill/exec/server/options/SystemOptionManager.java

```diff
@@ -314,6 +370,31 @@ public void deleteAllOptions(OptionType type) {
     }
   }
 
+  public void populateDefualtValues(Map<String, OptionValidator> VALIDATORS) {
+
+    // populate the options from the config
+    final Map<String, OptionValidator> tmp = new HashMap<>();
+    for (final Map.Entry<String, OptionValidator> entry : VALIDATORS.entrySet()) {
+
+      OptionValidator validator = entry.getValue();
+      final OptionValue.Kind kind = validator.getKind();
+      String name = entry.getValue().getOptionName();
+      OptionValue value;
+      String configPath = "drill.exec.options.";
```

Comment:
Let's pull the config path out to the top of the file as a constant. Better, as noted elsewhere, don't pass it in. Make it a constant in the base option validator class.